ci-operator/step-registry/baremetalds/sno/setup/baremetalds-sno-setup-commands.sh

```diff
@@ -42,6 +42,13 @@ then
   scp "${SSHOPTS[@]}" "${SHARED_DIR}/sno-additional-config" "root@${IP}:sno-additional-config"
 fi
 
+# Copy additional manifests
+if [[ -e "${SHARED_DIR}/manifest_*" ]]
+then
+  ssh "${SSHOPTS[@]}" "root@${IP}" "mkdir sno-additional-manifests"
+  scp "${SSHOPTS[@]}" "${SHARED_DIR}/manifest_*" "root@${IP}:sno-additional-manifests"
```

Comment:
`ipi-install-hosted-loki` will create manifest_... under ${SHARED_DIR}/ ?

Reply:
Yup - https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh#L29 and so on